functions.py

```diff
@@ -1,36 +1,35 @@
 import time
+
+def ReadLines(nameoffile):
+    with open(nameoffile,'r') as f:
+        arrayline = f.readlines()
+        return arrayline
+
 #how much is not taxed
 def SellWithoutTax():
-    f = open('new_register.txt','r')
-    try:
-        arrayline = f.readlines()
-        sum = 0.0
-        i = 0
+    arrayline = ReadLines('new_register.txt')
+    sum = 0.0
+    i = 0
+    arr = arrayline[i].split(',', 2)
+    while (time.time() - float(arr[1])) >= 31536000.0:

```

Comment:
lets have a global variable for 31536000.0 this number. 
Pls check what is a usual convention for globals. 
May be it should be _SECOND_IN_YEAR or just SECOND_IN_YEAR , I. dont remember.